### PR TITLE
fix: error message not displayed  on action failure

### PR DESCRIPTION
**Pull request by rustyb**
When `setFailed` is called when an `Error` is thrown the error message is not being displayed.

Currently on action fail the error is printed as `{}`
Screenshot 2021-10-01 at 11 30 01

When it should display the message of the thrown error: `"Invalid JSON string for input parameter release-rules."` 
Screenshot 2021-10-01 at 11 29 54

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -68,5 +69,6 @@ export const release = async (
 release()
   .then(reportResults)
   .catch((error: unknown): void => {
-    setFailed(JSON.stringify(error));
+    const finalError = handleMessageOrError(error);
```
Comment (mjroeleveld) [inline src/index.ts:72]: The function name does not completely represent what the function does, maybe name it `getSetFailedErrorString` and name the file `errors.ts`? Nit: there should be an <enter> below this line.

File: src/utilities/parseError.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { inspect } from 'util';
+
+/**
+ * Helper function to help the message part of the type Error
+ */
+
```
Comment (mjroeleveld) [inline src/utilities/parseError.ts:6]: We normally don't keep lines between the comment and function. ESLint should warn tho 🤔

Comment (rustyb) [reply src/utilities/parseError.ts]: Weird I didn't get warned by it. Have removed the line.

File: src/index.ts (hunk 1 above)
Comment (rustyb) [reply src/index.ts]: That's fine makes sense to rename the function. Sorry I don't understand the last point? Is it that there should be an empty line below this?

Comment (mjroeleveld) [reply src/index.ts]: Yess

**Review by mjroeleveld — APPROVED** [review]
Approve apart from the nit

**Pull request merged**